openapi-v1.yaml

```diff
@@ -2729,6 +2740,28 @@ definitions:
         type: string
         description: output location of the exported file
 
+  NotebookCopy:
+    description: Output information required to copy a notebook
+    type: object
+    properties:
+      output_uri:
+        type: string
+        description: output location of the TileDB File
+      name:
+        type: string
+        description: name to set for registered notebook
+
+  NotebookCopied:
```

Comment:
Again why don't we return `namespace, name` or a tiledb cloud uri?

Reply:
`name`, `namespace` added